sdk/nodejs/dynamic/index.ts

```diff
@@ -118,11 +118,55 @@ export interface UpdateResult<Outputs = any> {
     readonly outs?: Outputs;
 }
 
+/**
+ * {@link Config} is a bag of configuration values that can be passed to a provider's
+ * configure method.
+ *
+ * Use the Config.get and Config.require methods to retrieve a configuration
+ * value by key.
+ */
+export interface Config {
+    /**
+     *  get retrieves a configuration value by key. Returns None if the key is not present.
```

Comment:
```suggestion
     * get retrieves a configuration value by key. Returns undefined if the key is not present.
```
Not sure if there's any other Python wording we should tweak.

Reply:
🐍